src/components/views/messages/MPollBody.tsx

```diff
@@ -439,46 +441,50 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
     }
 }
 
-function renderEndedOption(
-    answer: IPollAnswer,
-    checked: boolean,
-    votesText: string,
-) {
+interface IEndedPollOptionProps {
+    answer: IPollAnswer;
+    checked: boolean;
+    votesText: string;
+}
+
+function EndedPollOption(props: IEndedPollOptionProps) {
     const classNames = `mx_MPollBody_endedOption${
-        checked ? " mx_MPollBody_endedOptionWinner": ""
+        props.checked ? " mx_MPollBody_endedOptionWinner": ""
     }`;
```

Comment:
can we use the classNames utility package here plase?

Reply:
Ah, I didn't know about that - thanks, done in 79e1d45d925270e2f65e2f1f030c7f502a938eaf